tests/system/small/test_remote_function.py

```diff
@@ -685,3 +685,54 @@ def test_read_gbq_function_enforces_explicit_types(bigquery_client, dataset_id):
         rf.read_gbq_function(
             str(neither_type_specified.reference), bigquery_client=bigquery_client
         )
+
+
+@pytest.mark.flaky(retries=2, delay=120)
+def test_df_apply_axis_1(session, scalars_dfs):
+    columns = ["bool_col", "int64_col", "int64_too", "float64_col", "string_col"]
+    scalars_df, scalars_pandas_df = scalars_dfs
+
+    def add_ints(row):
+        return row["int64_col"] + row["int64_too"]
```

Comment:
lets also have a test that uses some pandas behavior, such as getting the dtypes of the row, or getting the len of the row, or some statistical aggregate (eg skewness)

Reply:
in the large tests I am testing serialization of the row, which requires all the info to be present as it would be in pandas, could you PTAL if that suffices?

Reply:
I also added a (large) test for dtype and aggregates.